_posts/2020-07-12-ray-tune.md

```diff
@@ -0,0 +1,506 @@
+---
+layout: post
+title: Comparing Dask-ML and Ray Tune's Model Selection Algorithms
+tagline: Modern hyperparameter optimizations, Scikit-Learn support, framework support and scaling to many machines.
+author: <a href="https://stsievert.com">Scott Sievert</a> (University of Wisconsin–Madison)
+tags: [machine-learning, dask-ml, dask, ray]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+Hyperparameter optimization is the process of deducing model parameters that
+can't be learned from data. This process is often time- and resource-consuming,
+especially in the context of deep learning. A good description of this process
+can be found at "[Tuning the hyper-parameters of an estimator][tuning]," and
+the issues that arise are concisely summarized in Dask-ML's documentation of
+"[Hyper Parameter Searches]."
+
+[Hyper Parameter Searches]:https://ml.dask.org/hyper-parameter-search.html
+
+There's a host of libraries and frameworks out there to address this problem.
+[Scikit-Learn's module][skl-ms] has been mirrored [in Dask-ML][dml-ms] and
+[auto-sklearn], both of which offer advanced hyperparameter optimization
+techniques. Other implementations that don't follow the Scikit-Learn interface
+include [Ray Tune], [AutoML] and [Optuna].
+
+[AutoML]:https://www.automl.org/
+[auto-sklearn]:https://automl.github.io/auto-sklearn/master/
+
+[Ray] recently provided a wrapper to [Ray Tune] that mirrors the Scikit-Learn
+API. [The introduction][1] of this library states the following:
+
+[Ray]:https://docs.ray.io
+[Ray Tune]:https://docs.ray.io/en/master/tune.html
```

Comment:
```suggestion
[Ray] recently provided a wrapper to [Ray Tune] that mirrors [the Scikit-Learn API]. 
[The introduction][1] of this library states the following:

[Ray]:https://docs.ray.io
[Ray Tune]:https://docs.ray.io/en/master/tune.html
[the Scikit-Learn API]: https://github.com/ray-project/tune-sklearn
```

Just so that folks can see the code!